project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -1313,3 +1320,65 @@ function c4m_og_get_user_owned_groups($uid, $node_type) {
 
   return $nids;
 }
+
+/**
+ * Add the `Authoring information` tab to the group content editing forms.
+ *
+ * Allows group owner and administrators to change the author of content in the
+ * group. Limits possible authors to group members or anonymous user.
+ */
+function c4m_og_content_author(&$form) {
+  if (user_has_role('administrator')) {
```

Comment:
@ybaras 
Did you verify this one ?
because `user_has_role()` should get _rid_ and not the string representing the role name.
please use `user_role_load_by_name()` in order to get it